server/src/test/resources/de/frosner/broccoli/templates/curl-without-decription/template.conf

```diff
@@ -1,3 +1,4 @@
+documentation_url = "#curl-test-docs"
```

Comment:
Please change to `documentation-url`. For uniformity. For example: Look at `order-index`.